### PR TITLE
Add TFHub MobileNetV2 quantized model to benchmark

### PR DESCRIPTION
**Pull request by hcindyl**
This model represents a type of model without fused_activation_function, i.e., no RELU at the end of the ops. Put it in the benchmark to characterize the iree-compile fuse behavior for this kind of models.

### CODE REVIEW

File: build_tools/python/e2e_test_framework/models/tflite_models.py [hunk 1]
```diff
@@ -129,3 +129,14 @@
     "https://storage.googleapis.com/iree-model-artifacts/efficientnet_lite0_int8_2.tflite",
     entry_function="main",
     input_types=["1x224x224x3xui8"])
+
+MOBILENET_V2_INT8 = common_definitions.Model(
+    name="MobileNetV2_int8",
```
Comment (hcindyl) [inline build_tools/python/e2e_test_framework/models/tflite_models.py:134]: @pzread I would like to add the `id` field but I can't find how to define the ID if the model does not have a pre-exist run. Suggestions?

Comment (pzread) [reply build_tools/python/e2e_test_framework/models/tflite_models.py:134]: What do you mean not having a pre-exist run?

You can add a new id to `unique_ids.py` following others' format. Postsubmit will upload the new ids to dashboard.

Comment (hcindyl) [reply build_tools/python/e2e_test_framework/models/tflite_models.py:134]: Thanks. I add one line in build_tools/python/e2e_test_framework/models/README.md to explain how to generate the new uuid

**Comment by hcindyl** [comment]
Add this test to cover the model config described in https://github.com/openxla/iree/pull/12661

File: build_tools/python/e2e_test_framework/models/README.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ This directory contains the definitions of source models for e2e tests
     -   You can ask IREE team members for help if you don't have access.
 2.  Register a unique model ID in
     [build_tools/python/e2e_test_framework/unique_ids.py](/build_tools/python/e2e_test_framework/unique_ids.py).
+    -   Run uuid.uuid4() to generate a new uuid for the new model.
```
Comment (GMNGeoffrey) [inline build_tools/python/e2e_test_framework/models/README.md]: Note that you can also run `uuidgen` on Linux. We should probably explain more explicitly that these IDs are totally arbitrary artificial keys.

**Review by GMNGeoffrey — APPROVED** [review]

File: build_tools/python/e2e_test_framework/models/README.md (hunk 2 above)
Comment (hcindyl) [reply build_tools/python/e2e_test_framework/models/README.md]: Done.

**Review by pzread — APPROVED** [review]

**Pull request merged**